cachito/workers/pkg_managers/pip.py

```diff
@@ -27,6 +27,7 @@
     NetworkError,
     NexusError,
     ValidationError,
+    UploadError,
```

Comment:
Checked the CI failure, turns out this is in `cachito.workers.errors`, not `cachito.errors`